sqlserver/assets/configuration/spec.yaml

```diff
@@ -129,12 +129,226 @@ files:
         type: boolean
         example: false
         display_default: true
-    - name: include_ao_metrics
+    - name: database_metrics
       description: |
-        Include AlwaysOn availability group metrics.
-      value:
-        type: boolean
-        example: false
+        Configure the collection of database metrics
+      hidden: true
```

Comment:
Instead of making `database_metrics` hidden, can we hide individual options? 
For example, user was able to use legacy `include_ao_metrics` config option to turn on always on metrics. Ideally the new config option `database_metrics.ao_metrics` is visible to users in `conf.yaml.example`. 
`db_files_metrics` could be hidden because it is enabled by default.

Reply:
Sounds good, I can keep all the previously exposed ones non-hidden.